src/test/java/teammates/test/cases/storage/FeedbackResponsesDbTest.java

```diff
@@ -240,35 +240,36 @@ public void testGetFeedbackResponsesForQuestionInSection() {
 
         String questionId = fras.get("response1ForQ1S1C1").feedbackQuestionId;
 
-        List<FeedbackResponseAttributes> responses = frDb.getFeedbackResponsesForQuestionInSection(questionId, "Section 1");
+        List<FeedbackResponseAttributes> responses =
+                frDb.getFeedbackResponsesForQuestionInSection(questionId, "Section 1", null);
 
-        assertEquals(3, responses.size());
+        assertEquals(4, responses.size());
 
         ______TS("No responses as they are filtered out");
 
-        responses = frDb.getFeedbackResponsesForQuestionInSection(questionId, "Section 2");
+        responses = frDb.getFeedbackResponsesForQuestionInSection(questionId, "Section 2", null);
 
-        assertEquals(0, responses.size());
+        assertEquals(1, responses.size());
```

Comment:
Why does the number of responses returned increase here (and also the above test case)?